### PR TITLE
Add more information for running individual unit tests

### PR DESCRIPTION
**Pull request by m-kuhn**
@strk your recent comments made me look into this.

### CODE REVIEW

File: source/docs/developers_guide/unittesting.rst [hunk 1]
```diff
@@ -661,15 +661,59 @@ and ``-V`` to get verbose output:
 
   ## 0 tests passed, 1 tests failed out of 1
 
-  The following tests FAILED:
+  The following tests FAILED./output/bin/qgis_dxfexporttest:
```
Comment (m-kuhn) [inline source/docs/developers_guide/unittesting.rst]:
```suggestion
  The following tests FAILED:
```

File: source/docs/developers_guide/unittesting.rst [hunk 2]
```diff
@@ -665,11 +665,55 @@ and ``-V`` to get verbose output:
...
+  Config: Using QtTest library 5.12.5, Qt 5.12.5 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 9.2.1 20190827 (Red Hat 9.2.1-1))
+  PASS   : TestQgsDxfExport::initTestCase()
+  PASS   : TestQgsDxfExport::testPoints()
+  PASS   : TestQgsDxfExport::testLines()
+  ...
+  Totals: 19 passed, 4 failed, 0 skipped, 0 blacklisted, 612ms
+  ********* Finished testing of TestQgsDxfExport *********
+
+These tests also take `command line arguments
+<https://doc-snapshots.qt.io/qt5-5.9/qtest-overview.html#qt-test-command-line-arguments>`_,
```
Comment (havatv) [inline source/docs/developers_guide/unittesting.rst]:
```suggestion
<https://doc-snapshots.qt.io/qt5-5.9/qtest-overview.html#qt-test-command-line-arguments>`_.
```

File: source/docs/developers_guide/unittesting.rst [hunk 3]
```diff
@@ -665,11 +665,55 @@ and ``-V`` to get verbose output:
...
+this allows to run a specific subset of tests.
```
Comment (havatv) [inline source/docs/developers_guide/unittesting.rst]:
```suggestion
This makes it possible to run a specific subset of tests:
```

File: source/docs/developers_guide/unittesting.rst (hunk 2 above)
Comment (havatv) [inline source/docs/developers_guide/unittesting.rst]:
```suggestion
<https://doc-snapshots.qt.io/qt5-5.9/qtest-overview.html#qt-test-command-line-arguments>`_.
```

File: source/docs/developers_guide/unittesting.rst (hunk 3 above)
Comment (havatv) [inline source/docs/developers_guide/unittesting.rst]:
```suggestion
This makes it possible to run a specific subset of tests:
```

File: source/docs/developers_guide/unittesting.rst [hunk 4]
```diff
@@ -665,11 +665,55 @@ and ``-V`` to get verbose output:
...
 
+C++ Tests
+.........
+
 For C++ unit tests, QtCreator automatically adds run targets, so you can start
-them in the debugger.
+them from the debugger.
+
+If you go to *Projects* and there to the *Build & Run* --> Desktop *Run* tab, you can
+also specify command line parameters that will allow to run only a subset of the tests
```
Comment (havatv) [inline source/docs/developers_guide/unittesting.rst]:
```suggestion
also specify command line parameters that will allow a subset of the tests to be run
```

**Comment by havatv** [comment]
Thanks, @m-kuhn !

**Pull request merged**